crossword-xml-uploader/src/main/scala/com/gu/crossword/crosswords/CrosswordStore.scala

```diff
@@ -40,4 +40,11 @@ trait CrosswordStore {
     s3Client.deleteObject(config.crosswordsBucketName, awsKey)
   }
 
+  def archiveFailedCrosswordXMLFile(config: Config, awsKey: String): Unit = {
+    val processingFailedBucketName = "crossword-failed-files"
```

Comment:
```suggestion
    const processingFailedBucketName = "crossword-failed-files"
```
would also possibly suggest moving this out of the function

Reply:
this is pretty much identical to the `archiveCrosswordXMLFile` function in this file. This is Scala, not Typescript so it should be a `val`

Reply:
🤦  sorry, not sure how I missed that!